include/mbedtls/x509_crt.h

```diff
@@ -175,19 +175,34 @@ int mbedtls_x509_crt_parse_der( mbedtls_x509_crt *chain, const unsigned char *bu
                         size_t buflen );
 
 /**
- * \brief          Parse one or more certificates and add them
- *                 to the chained list. Parses permissively. If some
- *                 certificates can be parsed, the result is the number
- *                 of failed certificates it encountered. If none complete
- *                 correctly, the first error is returned.
+ * \brief          Parse one DER-encoded or multiple concatenated PEM-encoded
+ *                 certificates and add them to the chained list.
+ *
+ *                 For PEM-encoded CRTs, the function parses permissively:
+ *                 If at least one certificate can be parsed, the function
+ *                 returns the number of certificates for which parsing failed
+ *                 (hence \c 0 if all certificates were parsed successfully).
+ *                 If no certificate could be parsed, the function returns
+ *                 the first (negative) error encountered during parsing.
+ *
+ *                 PEM encoded certificates may be interleaved by other data
```

Comment:
I don't think we should explain the standard, but not a blocker

Reply:
I think this is helpful as it is not uncommon to see CRTs where the PEM data is prefixed by human readable data.

Reply:
OK, as I said, not a blocker.
I just think that sometimes informing too much only adds problems

Reply:
I agree with that, but here I think it removes problems instead of creating them.

Reply:
Personally, I don't think those lines add enough to merit being present.